### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -8,12 +8,83 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylon Labs, Ltd.
+Licensor:             Babylon Labs Ltd.
+
+Licensed Work:        Babylon Node
```
Comment (SebastianElvis) [inline LICENSE:13]: Does this need to be same as repo name?

**Review by SebastianElvis — APPROVED** [review]

File: LICENSE [hunk 2]
```diff
@@ -8,12 +8,83 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
...
+10. in any activity that transmits, exchanges, or is otherwise supported by the direct or indirect
+    proceeds of criminal or fraudulent activity.
+
+Any production use of the Licensed Work by you confirms your agreement to the foregoing limitations,
+and your understanding and agreement that they are limitations and not restrictions.
 
-Licensed Work:        Babylon
-                      The Licensed Work is (c) 2023 Babylon Labs, Ltd.
-
-Additional Use Grant: None.
```
Comment (gitferry) [inline LICENSE:88]: Is this empty line needed?

**Review by gitferry — APPROVED** [review]

**Review by Lazar955 — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (gitferry) [reply LICENSE:13]: with capitalized words?

Comment (vitsalis) [reply LICENSE:13]: Adam didn't like this being just "Babylon" as it might be confusing, so I added Babylon Node.

File: LICENSE (hunk 2 above)
Comment (vitsalis) [reply LICENSE]: good catch, nope

**Review by KonradStaniec — APPROVED** [review]

**Pull request merged**